Userland/Utilities/sql.cpp

```diff
@@ -22,7 +23,7 @@ class SQLRepl {
     {
         m_editor = Line::Editor::construct();
         m_editor->load_history(m_history_path);
-        
+
```

Comment:
The extra whitespace here and on lines 83 & 133 should be removed from the previous commit

Reply:
Did I tell you I hate merge conflicts? They will keep hounding me forever.